src/components/Persona/Persona.tsx

```diff
@@ -35,17 +36,19 @@ export class Persona extends React.Component<IPersonaProps, any> {
         ) }
 
         { presence !== PersonaPresence.none && <div className='ms-Persona-presence'></div> }
-
         { !hidePersonaDetails && (
-          <div className='ms-Persona-details'>
+          <div className= { personaDetailsClass }>
             <div className='ms-Persona-primaryText'>{ primaryText }</div>
             { secondaryText ? (
               <div className='ms-Persona-secondaryText'>{ secondaryText }</div>
             ) : (null) }
             <div className='ms-Persona-tertiaryText'>{ tertiaryText }</div>
             <div className='ms-Persona-optionalText'>{ optionalText }</div>
-          </div>) }
-      </div>
+            { innerControl ? (
+              <div className='ms-Persona-detailsWithInnerControl'>{ innerControl }</div>
```

Comment:
We need a different className here than what you use on the root div. We cannot assume they will always use the same styles. For example, right now you mark 'ms-Persona-detailsWithInnerControl' as display:none for some scenarios, you definitely do not want the root div to be display: none in those cases...